pkg/slo-controller/noderesource/plugins/batchresource/plugin.go

```diff
@@ -40,12 +42,18 @@ const PluginName = "BatchResource"
 
 var ResourceNames = []corev1.ResourceName{extension.BatchCPU, extension.BatchMemory}
 
+var Clock clock.Clock = clock.RealClock{} // for testing
+
 type Plugin struct{}
 
 func (p *Plugin) Name() string {
 	return PluginName
 }
 
+func (p *Plugin) Priority() int32 {
+	return framework.DefaultPluginPriority
```

Comment:
priority/seq can be overwrite in local repo, maybe defined all seq in an independent file.

Reply:
done. the ordering and registration are placed in `/noderesource/plugins_profile.go`